docs/code_contribution_guidelines.md

```diff
@@ -527,6 +530,16 @@ Here's an example replacing the `lightning-onion` version checked into `lnd` wit
  go mod edit -replace=github.com/lightningnetwork/lightning-onion@v0.0.0-20180605012408-ac4d9da8f1d6=github.com/roasbeef/lightning-onion@2e5ae87696046298365ab43bcd1cf3a7a1d69695
 ```
 
+<a name="LogLevels" />
+
+#### 4.10 Use of Log Levels
+
+There are six log levels available: `trace`, `debug`, `info`, `warn`, `error` and `critical`.
+
+Only use `error` for internal errors that are never expected to happen during
+normal operation. No event triggered by an external sources (rpc, chain backend,
```

Comment:
Is this correct though? In case of RPC error should we not log with error?

Reply:
That is indeed the intention of this definition. It is what this PR proposes.